types/recharts/index.d.ts

```diff
@@ -839,7 +839,7 @@ export interface LabelProps {
     position?: PositionType;
     children?: React.ReactNode[] | React.ReactNode;
     className?: string;
-    content?: React.ReactElement<any> | ContentRenderer<Label>;
+    content?: React.ReactElement<any> | ContentRenderer<any>;
```

Comment:
@mthmulders Same thing as before (not the component is being passed so I've changed it to `any`. We should add correct typings for the render functions eventually, but this works for now.